go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/dubbogo/net
 
-go 1.11
+go 1.17
```

Comment:
The go version 1.17 is too high to our project, maybe it could be downgraded to 1.16?